intern/service/boss/baseversion.py

```diff
@@ -181,8 +190,12 @@ def build_cutout_url(
         if len(id_list) > 0:
             urlWithParams += '?filter=' + self.convert_int_list_to_comma_sep_str(id_list)
 
-        if no_cache:
-            urlWithParams += '?no-cache=true'
+        if access_mode == 'no_cache':
+            urlWithParams += '?access-mode=no-cache'
```

Comment:
After changing the value of CacheMode.no_cache to 'no-cache', you can replace this entire if-elif-else block with `urlWithParams += '?access-mode={}'.format(access_mode)` See how the Enum makes it really easy to changes things in one place?